core/utility/ast-utils.ts

```diff
@@ -88,6 +93,26 @@ function _addSymbolToNgModuleMetadata(options: AddSymbolToNgModuleMetadataOption
   };
 }
 
+function _getChildrenRoutesPosition(source: ts.SourceFile): number | null {
+  const routes = findNodes(source, ts.SyntaxKind.VariableDeclarationList).find((item) => item.getText().includes('const routes: Routes'));
+  if (routes === undefined) {
+    return null;
+  }
+  const childrenRoutes = findNodes(routes, ts.SyntaxKind.PropertyAssignment).find((item) => item.getText().includes('children:'));
+  if (childrenRoutes === undefined) {
```

Comment:
@umbarutiba can we create the property `children` if it doesn't exist?

Reply:
as discussed